tests/e2e/instrumentation-nodejs-multicontainer/NAR.YAML

```diff
@@ -0,0 +1,361 @@
+apiVersion: v1
```

Comment:
Is this file intentional here? It does not seem to follow kuttl name conventions.

Reply:
No i remove it. Sorry.
Il also corrected the second it test of python-multicontainer